### DISCUSSION TITLE
IOTA token gone

### DISCUSSION THREAD

**Discussion by SaniMax20115** [iotaledger/firefly#6189] [Help & Support]
I took part in the last Shimmer STaking. Now I checked again on Firefly and my IOTA are gone. According to the history, no tokens have been sent from the wallet. Where did they go? Or does anyone have the same problem?

**Comment by zvikush** [comment]
I have the sort of similar issue: I installed a new firefly wallet in a new laptop and although I use the backup, I cannot see my iota coins (I can still see them on my old laptop). It seems something is wrong with the new wallet or I am not sure what is going on.

**Reply by Kane632** [reply]
I have almost the same issue. I formatted my PC and after installing today the Firefly app and recovering my wallet with the 24 word method I cannot see any balance.
